wopi.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import json
+import sys
+import datetime
+
+configfile = sys.argv[2] ##name of json file
+
+savepng = False
+if (len(sys.argv) > 3) and sys.argv[3] == "-s":
+    savepng = True
+    if len(sys.argv) > 4 :
+        if ((str(sys.argv[4]))[-4:] != ".png"): ##if name has no .png, add .png
+            savefile = sys.argv[4] + ".png"
+        else:
+            savefile = sys.argv[4]
+    else:
+        savefile = ("output " + str(datetime.datetime.now())[0:-7] + ".png") ##if name unspecified, default to output [time] .png
+
+with open(configfile,"r") as j:
+    joutput = json.load(j)
+
+valuenames = (np.array(joutput["axesNames"])) ##Names of the axes
+title = (joutput["title"])
+
+ax = plt.axes([0,0,1,1],projection = "polar")
```

Comment:
Also, I think the description for the extents in the first argument doesn't quite work with the polar plot. In my WOPI work I did something like `ax = plt.subplot(111, polar=True)` which seems to work better. I'm not entirely sure what is going on behind the scenes in matplotlib, but it seems to be able to find its own values when they arent specified.